README.md

```diff
@@ -32,8 +31,8 @@ I.) Create a dummy database, populated with data.
 
 Clone dmt-guide
 ````
-	$git clone https://github.com/esaude/dmt-guide
```

Comment:
I find it better with the indention as it was than without them.

Reply:
Okay! So you would prefer `$git ...` than `$ git ...`?

Reply:
Oh, i think you were right 😊

Reply:
Ohh great 👍